tests/integration/transformers/test_numerical.py

```diff
@@ -180,6 +181,21 @@ def test_nulls(self):
         reverse = bgmm_transformer.reverse_transform(transformed)
         np.testing.assert_array_almost_equal(reverse, data, decimal=1)
 
+    def test_all_nulls(self):
+        data = pd.DataFrame([np.nan, None] * 50, columns=['col'])
+        bgmm_transformer = BayesGMMTransformer()
+        bgmm_transformer.fit(data, list(data.columns))
+        transformed = bgmm_transformer.transform(data)
+
+        assert isinstance(transformed, pd.DataFrame)
+        assert transformed.shape == (100, 3)
+        assert all(x == 0.0 for x in transformed['col.normalized'])
+        assert all(x == 0.0 for x in transformed['col.component'])
+        assert all(x == 1.0 for x in transformed['col.is_null'])
```

Comment:
Could you make an expected DataFrame like
```
expected_transformed = pd.DataFrame({
    'col.normalized': [0] * 100,
    'col.component': [0] * 100,
    'col.is_null': [1.0] * 100
})
```
and just compare that transformed is equal to it?